fileio/ft_read_event.m

```diff
@@ -429,9 +429,10 @@
     event = read_ah5_markers(hdr, filename);
     
   case 'brainvision_vmrk'
-    fid=fopen(filename,'rt');
-    if fid==-1
-      ft_error('cannot open BrainVision marker file')
+    try
+      fid = fopen_or_error(filename,'rt');
+    catch err
+      ft_error(sprintf('cannot open BrainVision marker file: %s', err.message);
```

Comment:
this error would not happen right? It should be caught in the open_or_error, so the try-catch can be removed.

Reply:
I just checked, the error message in fopen_or_error suffices.